test-support/reference-trie/src/lib.rs

```diff
@@ -33,14 +33,13 @@ use trie_db::{
 use std::borrow::Borrow;
 use keccak_hasher::KeccakHasher;
 
-pub use trie_db::{
-	decode_compact, encode_compact,
-	nibble_ops, NibbleSlice, NibbleVec, NodeCodec, proof, Record, Recorder,
-	Trie, TrieConfiguration, TrieDB, TrieDBIterator, TrieDBMut, TrieDBNodeIterator, TrieError,
-	TrieIterator, TrieLayout, TrieMut,
+use trie_db::{
+	nibble_ops, NodeCodec,
+	Trie, TrieConfiguration,
+	TrieLayout, TrieMut,
 };
-pub use trie_root::TrieStream;
-pub mod node {
+use trie_root::TrieStream;
+mod node {
```

Comment:
I'm not a fan of re-exports as well, but this is a test-support crate, and the re-exports improve ergonomics for some doc tests

Reply:
I realize I misunderstood your comment (I did not put back the doc tests).

Reply:
but in fact it would also need to reestablished the cyclic dependency. Nevermind.